javaslang/src/main/java/javaslang/collection/Map.java

```diff
@@ -503,13 +503,13 @@ default int length() {
     @Override
     default <U> Seq<U> scanLeft(U zero, BiFunction<? super U, ? super Tuple2<K, V>, ? extends U> operation) {
         Objects.requireNonNull(operation, "operation is null");
-        return Collections.scanLeft(this, zero, operation, List.empty(), List::prepend, List::reverse);
+        return iterator().scanLeft(zero, operation).toStream();
```

Comment:
Because Map.scanRight() returns a Vector (and Vector is our default Seq), Map.scanLeft should also return Vector